### DISCUSSION TITLE
Error using RavenDB node client in edge script

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#19693] [Q&A] [answered]
I'm trying to use the node client in an edge script on Bunny.net. This is what I see in the log when I try to import it via https://esm.sh/ravendb. Is this a known issue?
<img width="867" alt="image" src="https://github.com/user-attachments/assets/1d4f7806-104e-4115-b3be-a4e306d4d079">
The Bunny support points me to this issue https://github.com/browserify/crypto-browserify/issues/213

**Comment by ml054** [comment, marked as answer]
Hi, RavenDB node client isn't supported on Edge network. 

More info can be found here: https://issues.hibernatingrhinos.com/issue/RDBC-863/Node.js-client-on-managed-hosting-providers

**Comment by marcuslindblom** [comment]
Oh, I see...
